src/locker/simple-locker.ts

```diff
@@ -0,0 +1,81 @@
+import type { exec, release } from 'lock';
+import { Lock, Locker } from "./locker.interface";
+
+//ILock from lock.d.ts misses isLocked string param
+interface ILock {
+	(key: string | string[], exec: exec): void;
+	isLocked(key: string): boolean;
+}
```

Comment:
because of [declaration merging](https://www.typescriptlang.org/docs/handbook/declaration-merging.html) you can extend the existing interface.